lib/Clustering.js

```diff
@@ -0,0 +1,263 @@
+'use strict'; // eslint-disable-line
+
+const cluster = require('cluster');
+
+class Clustering {
+    /**
+     * Constructor
+     *
+     * @param {number} size Cluster size
+     * @param {Logger} logger Logger object
+     * @param {number} [shutdownTimeout=5000] Change default shutdown timeout
+     * releasing ressources
+     * @return {Clustering} itself
+     */
+    constructor(size, logger, shutdownTimeout) {
+        this._size = size;
+        if (size < 1) {
+            throw new Error('Cluster size must be greater than or equal to 1');
+        }
+        this._shutdownTimeout = shutdownTimeout || 5000;
+        this._logger = logger;
+        this._shutdown = false;
+        this._workers = new Array(size).fill(undefined);
+        this._workersTimeout = new Array(size).fill(undefined);
+        this._workersStatus = new Array(size).fill(undefined);
+        this._status = 0;
+        this._exitCb = undefined; // Exit callback
+        this._index = undefined;
+    }
+
+    /**
+     * Function called after a stop() call
```

Comment:
Since these are all class methods, can you replace `Function` in the descriptions to `Method`?